packages/coding-standard/src/Rules/PrefferedMethodCallOverFuncCallRule.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Rules;
+
+/**
+ * @see \Symplify\CodingStandard\Tests\Rules\PrefferedMethodCallOverFuncCallRule\PrefferedMethodCallOverFuncCallRuleTest
+ */
+final class PrefferedMethodCallOverFuncCallRule extends AbstractPrefferedCallOverFuncRule
+{
+    /**
+     * @var string
+     */
+    public const ERROR_MESSAGE = 'Use "%s::%s()" method call over "%s()" func call';
```

Comment:
```suggestion
    public const ERROR_MESSAGE = 'Use "%s->%s()" method call over "%s()" func call';
```

This is more clear that is not a static call

Reply:
updated.